graphdatascience/procedure_surface/cypher/catalog_cypher_endpoints.py

```diff
@@ -99,13 +105,50 @@ def filter(
 
     @property
     def sample(self) -> GraphSamplingEndpoints:
-        """Get graph sampling endpoints.
-
-        Returns:
-            GraphSamplingEndpoints: Graph sampling endpoints for cypher implementation.
-        """
         return GraphSamplingCypherEndpoints(self._query_runner)
 
+    def generate(
+        self,
+        graph_name: str,
+        node_count: int,
+        average_degree: float,
+        *,
+        relationship_distribution: Optional[str] = None,
+        relationship_seed: Optional[int] = None,
+        relationship_property: Optional[RelationshipPropertySpec] = None,
+        orientation: Optional[str] = None,
+        allow_self_loops: Optional[bool] = None,
+        read_concurrency: Optional[int] = None,
+        job_id: Optional[str] = None,
+        sudo: Optional[bool] = None,
+        log_progress: Optional[bool] = None,
+        username: Optional[str] = None,
+    ) -> GraphGenerationStats:
+        config = ConfigConverter.convert_to_gds_config(
+            relationship_distribution=relationship_distribution,
+            relationship_seed=relationship_seed,
+            relationship_property=relationship_property.model_dump(by_alias=True) if relationship_property else None,
+            orientation=orientation,
+            allow_self_loops=allow_self_loops,
+            read_concurrency=read_concurrency,
+            job_id=job_id,
+            sudo=sudo,
+            log_progress=log_progress,
+            username=username,
```

Comment:
I recall commenting on this on the last PR. so now these are all snake_case, and then the converter itself makes them camelCase. so it looks correct.